README.md

```diff
@@ -216,6 +216,36 @@ By default: `['GET', 'HEAD', 'OPTIONS', 'TRACE' ]`
 
 This plugin will always retry on 503 errors, _unless_ `retryMethods` does not contain `GET`.
 
+#### `globalAgent`
+
+Enables the possibility to explictly opt-in for global agents.
+
+Usage for undici global agent:
+```js
+import { setGlobalDispatcher, ProxyAgent } from 'undici'
+
+const proxyAgent = new ProxyAgent('my.proxy.server')
+setGlobalDispatcher(proxyAgent)
+
+fastify.register(FastifyReplyFrom, {
+  base: 'http://localhost:3001/',
+  globalAgent: true, 
+  undici: {
+  }
```

Comment:
why is this empty object needed? It's not user friendly.

Reply:
I believe `undici: {  }` can be removed